main.go

```diff
@@ -104,40 +141,83 @@ func elbDNSName(c *conf) {
 		log.Fatal(err)
 	}
 
-	c.elbName = *resp.LoadBalancerDescriptions[0].DNSName
+	return *resp.LoadBalancerDescriptions[0].DNSName
 }
 
-func destroyDNS(c *conf, domain string, hc *http.Client) error {
-	body := fmt.Sprintf("{\"zone\": \"ft.com\", \"name\": \"%s\"}", domain)
-	req, err := http.NewRequest("DELETE", c.konsDNSEndPoint, strings.NewReader(body))
+func getCurrentCNAME(c *conf, domain string) (string, error) {
```

Comment:
Would it be worth to extract the common code from the update and create operations? The only differences are in the body and request (and a log).